src/Missings.jl

```diff
@@ -514,4 +514,32 @@ julia> emptymissing(first)([1], 2)
 """
 emptymissing(f) = (x, args...; kwargs...) -> isempty(x) ? missing : f(x, args...; kwargs...)
 
+# Variant of `isless` where `missing` is the smallest value
```

Comment:
This doesn't describe `missingsmallest`. Not really useful anyway since we have a docstring.
```suggestion
```